repository/github_user.go

```diff
@@ -0,0 +1,96 @@
+//Package repository container the fuctions to handle githubusers model information
+package repository
+
+import (
+	"encoding/csv"
+	"io"
+	"log"
+	"lolidelgado/github-users/models"
+	"os"
+	"strconv"
+)
+
+type GithubUser struct {
+	githubUserCsvFileName string
+}
+
+type IGithubUserRepository interface {
+	FetchAll() ([]models.GithubUser, error)
+}
+
+const staticFilesPath = "static/"
+
+func NewGithubUser(fileName string) *GithubUser {
+	return &GithubUser{
+		fileName,
+	}
+}
+
+func (g *GithubUser) FetchAll() ([]models.GithubUser, error) {
+	lines, err := g.readCsv()
+	if err != nil {
+		return nil, err
+	}
+	return arrayToGithubUser(lines), nil
+}
+
+func (g *GithubUser) GetById(id int) ([]models.GithubUser, error) {
```

Comment:
Question, if you are getting a single GithubUser, why your return here is an array?

As mentioned before, this should be named `GetByID`.